packages/stryker-babel/src/BabelTranspiler.ts

```diff
@@ -0,0 +1,87 @@
+import { Transpiler, TranspilerOptions, TranspileResult, FileLocation } from 'stryker-api/transpile';
+import { File, TextFile, FileKind } from 'stryker-api/core';
+import * as babel from 'babel-core';
+import * as path from 'path';
+import { EOL } from 'os';
+import { CONFIG_KEY_OPTIONS } from './helpers/keys';
+
+class BabelTranspiler implements Transpiler {
+  private babelConfig: babel.GeneratorOptions;
+  private knownExtensions: string[];
+
+  public constructor(options: TranspilerOptions) {
+    this.babelConfig = options.config[CONFIG_KEY_OPTIONS];
+
+    this.knownExtensions = ['.js', '.jsx', '.ts'];
```

Comment:
Where did you find this list? Why would typescript be allowed? If that is the case, maybe we also need all these extensions: https://github.com/Microsoft/TypeScript/blob/0b9e4987f477f46e66848a08e8f3db38338c84e0/src/compiler/types.ts#L4247

Reply:
I removed typescript from the list